src/System.Private.CoreLib/shared/Interop/Windows/BCrypt/Interop.BCryptGenRandom.cs

```diff
@@ -19,19 +19,25 @@ internal static unsafe int BCryptGenRandom(byte* pbBuffer, int count)
         }
 
         private const int BCRYPT_USE_SYSTEM_PREFERRED_RNG = 0x00000002;
-        internal const int STATUS_SUCCESS = 0x0;
-        internal const int STATUS_NO_MEMORY = unchecked((int)0xC0000017);
+
+        internal enum NTSTATUS : uint
```

Comment:
This is used by other BCrypt files as well, so it should not be folded into this file.

Reply:
I am just applying https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/interop-guidelines.md :

If structs/constants will be used on their own without an associated DllImport, or if they may be used with multiple DllImports not in the same file, they should be declared in a separate file.

Reply:
aaahhh ok! thank's i missed it!

Reply:
Yes, it is. (The failures in the current run will go away on rerun - coreclr/corefx was out of sync temporarily.)

Reply:
Great thank's!